### PR TITLE
Add com.github.weclaw1.ImageRoll

### PR DESCRIPTION
**Pull request by weclaw1**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.weclaw1.ImageRoll.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+app-id: com.github.weclaw1.ImageRoll
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```
Comment (hfiguiere) [inline com.github.weclaw1.ImageRoll.yaml:3]:
```suggestion
runtime-version: '40'
```

File: com.github.weclaw1.ImageRoll.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hfiguiere) [inline com.github.weclaw1.ImageRoll.metainfo.xml:1]: why adding this file since it's installed from the sources

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, buld com.github.weclaw1.ImageRoll

**Comment by hfiguiere** [comment]
bot, build com.github.weclaw1.ImageRoll

File: com.github.weclaw1.ImageRoll.yaml (hunk 1 above)
Comment (weclaw1) [reply com.github.weclaw1.ImageRoll.yaml]: Fixed

**Comment by LeandroStanger** [comment]
bot, build com.github.weclaw1.ImageRoll

**Pull request closed without merging**